src/Bijectors.jl

```diff
@@ -5,8 +5,9 @@ using Reexport, Requires
 using StatsFuns
 using LinearAlgebra
 using MappedArrays
+using Tracker: param
```

Comment:
I think we should be able to get rid of the dependency on `Tracker.pram` by making the type generic. I.e. we could initialize planar flow by `Arrays` or `Params` depending on the user. But I think this is less urgent.